lms/static/sass/application-rtl.scss.mako

```diff
@@ -28,72 +28,7 @@
   @import '${env.get('THEME_NAME')}';
 % endif
 
-@import 'base/base';
```

Comment:
Yup, agreed. Once we move away from mako variables, we can add in the rest of the import to the build file.